sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -268,7 +332,7 @@ export interface UpdateKeyOptions {
     expires?: Date;
```

Comment:
`expiresOn`

Reply:
This won't be done as part of this PR. The issue tracking this change is #5577